CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.6.2] - 2023-08-07
```

Comment:
As 0.6.2 has been released, this should be included under the "Unreleased" section instead.